clang/test/SemaHLSL/BuiltIns/fmod-errors.hlsl

```diff
@@ -0,0 +1,38 @@
+
+// RUN: %clang_cc1 -finclude-default-header -triple dxil-pc-shadermodel6.6-library %s -fnative-half-type -emit-llvm-only -disable-llvm-passes -verify -verify-ignore-unexpected
+
+float test_too_few_arg() {
```

Comment:
These test cases are fine, but the only two  I was expecting was that `double` exclusion error  defined via `CheckFloatOrHalfRepresentations`  and the # of function args check defined in `clang/lib/Sema/SemaHLSL.cpp`.

Seems like the double exclusion is missing.

Reply:
Done. Add 2 double type tests